Cargo.toml

```diff
@@ -18,7 +18,7 @@ serde_json = "1.0"
 reqwest = "~0.6"
 log = "0.3"
 quick-error = "1.1.0"
-url = "1.4"
+url = "1.5"
```

Comment:
We shouldn't need to increment this dependency. We don't require anything from url `1.5`.